[ccfits] doesn't support single config builds

```
Building ccfits[core]:x64-windows...
-- Downloading https://heasarc.gsfc.nasa.gov/fitsio/CCfits-2.5/CCfits-2.5.tar.gz -> CCfits-2.5.tar.gz...
-- Extracting source D:/downloads/CCfits-2.5.tar.gz
-- Applying patch dll_exports.patch
-- Applying patch fix-dependency.patch
-- Using source at D:/buildtrees/ccfits/src/CCfits-2-de1fe204e9.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
-- Building x64-windows-rel
CMake Error at ports/ccfits/portfile.cmake:25 (file):
  file RENAME failed to rename

    D:/packages/ccfits_x64-windows/debug/lib/CCfits.dll

  to

    D:/packages/ccfits_x64-windows/debug/bin/CCfits.dll

  because: The system cannot find the path specified.

  
```

--- a/ports/ccfits/portfile.cmake
+++ b/ports/ccfits/portfile.cmake
@@ -28,7 +28,7 @@ if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
 endif()
 
 # Remove duplicate include files
-# file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 # Patch installed headers to look in the correct subdirectory
 file(GLOB HEADERS "${CURRENT_PACKAGES_DIR}/include/CCfits/*")

--- a/ports/ccfits/portfile.cmake
+++ b/ports/ccfits/portfile.cmake
@@ -20,13 +20,15 @@ vcpkg_cmake_install()
 
 if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin")
-    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
     file(RENAME "${CURRENT_PACKAGES_DIR}/lib/CCfits.dll" "${CURRENT_PACKAGES_DIR}/bin/CCfits.dll")
-    file(RENAME "${CURRENT_PACKAGES_DIR}/debug/lib/CCfits.dll" "${CURRENT_PACKAGES_DIR}/debug/bin/CCfits.dll")
+    if(NOT VCPKG_BUILD_TYPE)
+        file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
+        file(RENAME "${CURRENT_PACKAGES_DIR}/debug/lib/CCfits.dll" "${CURRENT_PACKAGES_DIR}/debug/bin/CCfits.dll")
+    endif()
 endif()
 
 # Remove duplicate include files
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+# file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 # Patch installed headers to look in the correct subdirectory
 file(GLOB HEADERS "${CURRENT_PACKAGES_DIR}/include/CCfits/*")
@@ -38,4 +40,4 @@ vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/include/CCfits/CCfits.h
     "#include \"longnam.h\"" "#include \"cfitsio/longnam.h\""
 )
 
-file(INSTALL "${SOURCE_PATH}/License.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/License.txt")

--- a/ports/ccfits/vcpkg.json
+++ b/ports/ccfits/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "ccfits",
   "version": "2.5",
-  "port-version": 10,
+  "port-version": 11,
   "description": "CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++.",
   "homepage": "https://heasarc.gsfc.nasa.gov/fitsio/CCfits/",
   "license": "NASA-1.3",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1406,7 +1406,7 @@
     },
     "ccfits": {
       "baseline": "2.5",
-      "port-version": 10
+      "port-version": 11
     },
     "cctag": {
       "baseline": "1.0.2",

--- a/versions/c-/ccfits.json
+++ b/versions/c-/ccfits.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d42ea5328ac404ea3f6ce45a20af3836acab13c7",
+      "version": "2.5",
+      "port-version": 11
+    },
+    {
       "git-tree": "77b570d63ebf64b4110d2555ce945144cc695364",
       "version": "2.5",
       "port-version": 10

--- a/versions/c-/ccfits.json
+++ b/versions/c-/ccfits.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d42ea5328ac404ea3f6ce45a20af3836acab13c7",
+      "git-tree": "69b00d0813b3ec9b7b963f07cf570cca5a8e2fd5",
       "version": "2.5",
       "port-version": 11
     },

